cfme/tests/services/test_rest_services.py

```diff
@@ -35,6 +36,7 @@
 pytestmark = [
     pytest.mark.long_running,
     test_requirements.service,
+    test_requirements.rest,
     pytest.mark.tier(2),
```

Comment:
Question:
Can we add two ```test_requirements``` globally?

Reply:
I saw more than 1 requirements were added in many places which is why I added it here as well, since we're testing services via REST.